affinity/cpp-20/d0796r3.md

```diff
@@ -264,27 +280,27 @@ The `execution_context` class provides an abstraction for managing a number of l
 Below *(Listing 5)* is an example of how this extended interface could be used to construct an *execution context* from an *execution resource* which is retrieved from the *system’s resource topology*. Once an *execution context* is constructed it can then still be queried for its *execution resource*, and that *execution resource* can be further partitioned.
 
 ```cpp
-auto &resources = execution::this_system::get_resources();
+auto systemResource = std::this_system::discover_topology();
 
-execution::execution_context execContext(resources[0]);
+execution::execution_context execContext(systemResource[0]);
 
-auto &systemLevelResource = execContext.resource();
+auto &execResource = execContext.resource();
 
-// resource[0] should be equal to execResource
+// systemResource[0] should be equal to execResource
 
-for (auto res : systemLevelResource.resources()) {
-    std::cout << res.name()  << `\n`;
+for (const execution::execution_resource res : execResource()) {
```

Comment:
extra () at the end of execResource,
also why const but no &?

Reply:
Good catch, thanks.